mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.erb

```diff
@@ -2587,6 +2587,328 @@ func TestAccComputeInstance_regionBootDisk(t *testing.T) {
 	})
 }
 
+<% unless version == 'ga' -%>
```

Comment:
Thanks for adding the test cases. Is there a test for removing the security policies?

Reply:
I added a new step which I clean the policies and check if they don´t exist on the "testAccComputeInstance_nic_securityPolicyCreateWithTwoNicsAndTwoAccessConfigsUpdateSwapPolicies" test.

Reply:
Thanks for adding it.